### PR TITLE
Update security-context.md

### PR DESCRIPTION
**Pull request by bertreyking**
Hello, Chinese document [Pod 或容器配置安全上下文](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/security-context/), link section Pod 安全性准入  404 Page not found

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 1]
```diff
@@ -849,7 +849,7 @@ kubectl delete pod security-context-demo-4
 * [使用最新的安全性增强来调优 Docker（英文）](https://github.com/containerd/containerd/blob/main/docs/cri/config.md)
 * [安全上下文的设计文档（英文）](https://github.com/kubernetes/design-proposals-archive/blob/main/auth/security_context.md)
 * [属主管理的设计文档（英文）](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/volume-ownership-management.md)
-* [Pod 安全性准入](zh-cn/docs/concepts/security/pod-security-admission/)
+* [Pod 安全性准入](https://kubernetes.io/zh-cn/docs/concepts/security/pod-security-admission/)
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:852]:
```suggestion
* [Pod 安全性准入](/zh-cn/docs/concepts/security/pod-security-admission/)
```
Original is correct.

**Review by windsonsea — COMMENTED** [review]
Thanks for your contribution. You can run the following commands in your local repo:
```
./scripts/lsync.sh content/zh-cn/docs/tasks/configure-pod-container/security-context.md
```
to find some differences (if any) between en and zh:

image

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/tasks/configure-pod-container/security-context.md:852]: Right, a leading `\` is missing. That is all.

Comment (bertreyking) [reply content/zh-cn/docs/tasks/configure-pod-container/security-context.md:852]: Changed.

**Review by bertreyking — COMMENTED** [review]
changed.

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md (hunk 1 above)
Comment (windsonsea) [reply content/zh-cn/docs/tasks/configure-pod-container/security-context.md:852]: It's still not correct. The green line is my suggestion. You can directly copy/paste the suggested change if you accept it.

**Comment by windsonsea** [comment]
It's still not correct. The green line is my suggestion. You can directly copy/paste the suggested change if you accept it.

If you don't accept, don't mark it as `resolved`.

**Pull request merged**